crates/wastebin_core/src/db.rs

```diff
@@ -502,6 +511,19 @@ impl Handler {
         Ok(())
     }
 
+    fn delete_many(&mut self, ids: Vec<Id>) -> Result<usize, Error> {
+        let tx = self.conn.transaction()?;
+
+        let mut affected = 0;
+
+        for id in ids {
+            affected += tx.execute("DELETE FROM entries WHERE id=?1", params![id.to_i64()])?;
+        }
```

Comment:
Why do you count it like that and not as `ids.len()`? At the end you always just affect a single row because ID is a primary key.